test/import/FSHImporter.Alias.test.ts

```diff
@@ -22,7 +22,8 @@ describe('FSHImporter', () => {
       Alias: UCUM = http://unitsofmeasure.org
       `;
 
-      const result = importText(input);
+      const results = importText([new FileInfo(input)]);
+      const result = results[0];
```

Comment:
Seeing as this is a common pattern in the importer tests, you could simplify it by creating a helper function something like:
```ts
export function importSingleText(content string, path? string): FSHDocument {
  return importText([new FileInfo(content, path)])[0];
}
```
Then in each test, you can go back to the simpler statement:
```ts
const result = importSingleText(input);
```